spec/integration/guest_view_spec.rb

```diff
@@ -47,7 +51,6 @@
 
     it "shows the loaded image for each item" do
       create_item
-
```

Comment:
Shouldn't this be in there? Action: create_item. Exercise: Visit menu_path. Expectation: expect...